tests/Unit/Http/UrlsTest.php

```diff
@@ -7,24 +7,15 @@
 
 final class UrlsTest extends BaseTestCase
 {
-    public function testSimpleUrlsAreReturned()
+    public function testSanitize()
     {
-        $this->assertSame('some_url.com/path/', Urls::sanitize('some_url.com/path/'));
```

Comment:
All this use cases have been moved to https://github.com/DataDog/dd-trace-php/pull/1396/files#diff-11c158ecb066ee67275f17d6c2af030ba5e43772caf48122c9e7d1ad556eab82